docs/extra/logs-collect.md

```diff
@@ -0,0 +1,23 @@
+# Collecting logs for Support
+
+To help troubleshoot issues, obtain the logs from your Codacy instance and send them to Codacy's Support:
+
+1.  Collect logs from your Codacy instance using Fluentd:
+
+    ```bash
+    --set fluentdoperator.enabled=true
```

Comment:
What is the full command to use here? Is it the one below?

`helm upgrade codacy codacy/codacy --set fluentdoperator.enabled=true`

If possible we should provide the full command here.

Reply:
We should not have the full command here. It will get outdated pretty quickly.
I see your idea but it will outdated outdated.
You can put a link to the installation guide.

Reply:
And this field is now true by default, so I doubt that any client needs to change it at this point.

Reply:
Indeed, the REAME.md file states that `fluentdoperator.enable` is set to True by default, with a log expiration date of 7 days. This is great because it means I can simplify the instructions. :smile: